pos_frequency_pipeline.py

```diff
@@ -1,8 +1,61 @@
 """
 Implementation of POSFrequencyPipeline for score ten only.
 """
+import json
+import re
+
+from article import Article
+from constants import ASSETS_PATH
+from collections import Counter
+from pipeline import CorpusManager
+from pymystem3 import Mystem
+from visualizer import visualize
 
 
 class POSFrequencyPipeline:
     def __init__(self, assets):
-        pass
+        self.assets = assets
+        self.raw_text = ''
+        self.freq_list = []
+
+    def run(self):
+        numbers = self.assets.get_articles()
+        for number in numbers:
+            article = Article(url=None, article_id=number)
+            self.raw_text = article.get_raw_text()
+            self.freq_list = self._process()
+            self._write_to_meta(number)
+            visualize(statistics=self.freq_list, path_to_save=ASSETS_PATH + f'/{number}_image.png')
+
+    def _process(self):
+        """
+        Performs processing of each text
+        """
+        result = Mystem().analyze(self.raw_text)
+        list_of_pos = []
+        for stem_dict in result:
```

Comment:
POSFrequencyPipeline does not perform text analysis - it should work only on top of the existing processed files (!)